inline_scan.sh

```diff
@@ -140,6 +143,11 @@ get_and_validate_analyzer_options() {
 
     SYSDIG_SCANNING_URL="${SYSDIG_BASE_SCANNING_URL}"/api/scanning/v1
     SYSDIG_ANCHORE_URL="${SYSDIG_SCANNING_URL}"/anchore
+
+    if [[ "${l_flag:-}" ]]; then
+        CURL_AUTH='--basic'
```

Comment:
So we use either --basic or -k (insecure) option? Shouldn't they be independent?